terminus-ui/popover/src/popover.component.scss

```diff
@@ -0,0 +1,77 @@
+@import './../../scss/helpers/color';
+@import './../../scss/helpers/shadows';
+@import './../../scss/helpers/spacing';
+
+$PLACEMENT_MARGIN: 5px;
```

Comment:
I don't believe SCSS files get isolated scope, so we should probably move this variable just inside the first SCSS scope (in this instance `popover__container`).